Task 6/Task6.4/main.css

```diff
@@ -0,0 +1,45 @@
+body{
+    background-color: rgb(46, 70, 94);
+}
+.hosting-block
+{
+background-color:rgb(255, 255, 255);
+box-shadow: 0px 5px 50px ;
+width: 700px;
+text-align:  center;
+border-radius: 15px;
+padding: 0;
+margin: 0;
+
+}
+.hosting-block > div {
+    display: inline-block;
+    padding: 2%;
+    margin: 2% 0%;
+}
+
+
+.hosting-block > div:last-child {
+border-left:slategray solid 1px;
+}
+.hosting-block > div:first-child {
+    border-right:slategray solid 1px;
+    }
```

Comment:
This stops working when you have more than 3 elements. Better to have some border on **all** elements **except** one at the start/end